relationships.md

```diff
@@ -0,0 +1,324 @@
+              },
+              "rightType": {
+                "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/4"
+              },
+              "self": {
+                "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/5"
+              }
+            },
+            "_embedded": {
+              "leftType": {
+                "id": 2,
+                "label": "Person",
+                "type": "entitytype",
+                "_links": {
+                  "self": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+                  },
+                  "relationshiptypes": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2/relationshiptypes"
+                  }
+                }
+              },
+              "rightType": {
+                "id": 4,
+                "label": "OrgUnit",
+                "type": "entitytype",
+                "_links": {
+                  "self": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/4"
+                  },
+                  "relationshiptypes": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/4/relationshiptypes"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      …
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/relationships/search/byLabel?label=isPersonOfOrgUnit"
+    }
+  },
+  "page": {
+    "number": 0,
+    "size": 20,
+    "totalPages": 1,
+    "totalElements": 2
+  }
+}
+```
+
+This is similar to
+https://dspace7-internal.atmire.com/rest/#/rest/api/core/communities/search/subCommunities?parent=daa2657d-5f39-4876-9536-ace42e96b440
+
+It embeds all relationships where the relationship type has the given label on either the left or the right label
+
+
+This can be further filtered to a single DSO using 
+https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/relationships/search/byLabel?label=isPersonOfOrgUnit&dso=f2235aa6-6fe7-4174-a690-598b72dd8e44 which contains all relationships created using the relationship type isPersonOfOrgUnit for which one item is f2235aa6-6fe7-4174-a690-598b72dd8e44
+The dso parameter is optional
+
+## Deleting a relationship
+
+**DELETE /api/core/relationships/<:id>**
+
+Delete a relationship between 2 items.
+
+A sample CURL command would be:
+```
+curl -D - -XDELETE 'https://dspace7-entities.atmire.com/rest/api/core/relationships/890'  -H 'Authorization: Bearer eyJhbGciO…'
+```
+
+* 204 No content - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions
+* 404 Not found - if the item doesn't exist (or was already deleted)
```

Comment:
small typo here, should be the relationship doesn't exist

Reply:
This has been fixed